src/main/java/org/terasology/signalling/componentSystem/SignalSystem.java

```diff
@@ -319,7 +357,12 @@ private boolean hasSignalForOr(Collection<NetworkSignals> networkSignals) {
         return false;
     }
 
-    //
+    /**
+     * Passes on the most powerful of the signals in the given network. All of them if one signal has infinite strength.
```

Comment:
"All of them if one signal has infinite strength"
All of them what? The sentence is imcomplete.
Maybe you meant "passes on all of them .." ?